src/components/Transactions/TransactionDetail.tsx

```diff
@@ -38,7 +38,7 @@ export default function TransactionDetail({
       <Stack spacing={2} mt={3} width={'100%'}>
         <Box>
           <Typography>
-            {t('app.transactionDetail.createdOn') + ' ' + transactionData.dateTime.toDateString()}
+            {`${t('app.transactionDetail.createdOn')} ${transactionData.dateTime}`}
```

Comment:
![Screenshot 2024-05-08 alle 15 13 01](https://github.com/pagopa/pagopa-arc-fe/assets/1122389/f4acf82c-0826-4504-9ccd-a53b08aeb3f2)
please, use [this function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) to have a better render of date